nextcord/types/guild.py

```diff
@@ -148,3 +150,30 @@ class ChannelPositionUpdate(TypedDict):
 class RolePositionUpdate(TypedDict):
     id: Snowflake
     position: NotRequired[Optional[Snowflake]]
+
+
+class OnboardingPromptOption(TypedDict):
+    id: Snowflake
+    channel_ids: List[Snowflake]
+    role_ids: List[Snowflake]
+    emoji: Emoji
```

Comment:
This seems to be `NotRequired`
![image](https://github.com/nextcord/nextcord/assets/34910574/e1c713cf-3321-4d92-bb11-05d1f31e5827)